CHANGELOG.md

```diff
@@ -3,6 +3,37 @@ Changelog
 
 `0.4.0` (Unreleased)
 
+* Filter by message attributes (#66) 
+* All Relé settings are defined in a dict (#60)
+
+Old structure:
+```python
+from google.oauth2 import service_account
+RELE_GC_CREDENTIALS = service_account.Credentials.from_service_account_file(
+    'rele/settings/dummy-credentials.json'
+)
+RELE_GC_PROJECT_ID = 'dummy-project-id'
+```
+
+New structure:
+```python
+from google.oauth2 import service_account
+RELE = {
+    'GC_CREDENTIALS': service_account.Credentials.from_service_account_file(
+        'rele/settings/dummy-credentials.json'
+    ),
+    'GC_PROJECT_ID': 'dummy-project-id',
+    'MIDDLEWARE': [
+        'rele.contrib.DjangoDBMiddleware',
+        'rele.contrib.LoggingMiddleware',
+    ],
+    'SUB_PREFIX': 'delivery',
+    'APP_NAME': 'delivery',
+}
+```
+* `rele.contrib.middleware` (#55)
```

Comment:
I miss the customization of the ack deadline (#44). Did you decide it wasn't worth mentioning it? 🤔

Reply:
🤦‍♂ There were so many, I was bound to miss one!